src/System.Security.Cryptography.Xml/tests/SignedXmlTest.cs

```diff
@@ -308,7 +308,7 @@ public void AsymmetricDSASignature()
 
             Assert.Null(signedXml.SignatureLength);
             Assert.Equal(SignedXml.XmlDsigDSAUrl, signedXml.SignatureMethod);
-            Assert.Equal(40, signedXml.SignatureValue.Length);
+            Assert.True(signedXml.SignatureValue.Length >= 40, $"At {signedXml.SignatureValue.Length} bytes long, the signature value is less than the minimum length of 40 bytes.");
```

Comment:
The message in these asserts prints thusly:

```
At 30 bytes long, the signature value is less than the minimum length of 40 bytes.
Expected: True
Actual: False
```

Note the Expected and Actual, and how they're now wrong for the message.

`Assert.InRange(signedXml.SignatureValue.Length, 40, int.MaxValue);`, perhaps?